ConferenceV2/app/services/singletons/conference_call_manager.py

```
@@ -29,18 +31,37 @@ def __init__(
+                conf_id,
+                ws_url=f"{self.ws_base_url}?id={conf_id}",
+            ),
+            connection_manager=self.smartphone_connection_manager_factory.create(
+                self.smartphone_connection_manager_type, conf_id
+            ),
+            storage_manager=self.storage_manager,
         )
+        self._attach_redis(conference_call)
+        return conference_call
+
+    async def create_conference(self, teacher_phone: str, student_phones: List[str], leader_phone: str = None, teacher_name: str | None = None, student_names: List[str] | None = None) -> ConferenceCall:
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)